cli/command/load_balancer.go

```diff
@@ -10,6 +17,447 @@ func NewLoadBalancerCommand(b *CommandBase) *LoadBalancerCommand {
-func (e *LoadBalancerCommand) Command() cli.Command {
-	return cli.Command{}
+func (l *LoadBalancerCommand) Command() cli.Command {
+	defaultPortFlag := cli.StringSlice([]string{"80:80/tcp"})
+	return cli.Command{
+		Name:  "loadbalancer",
+		Usage: "Manage layer0 Load Balancers",
+		Subcommands: []cli.Command{
+			{
+				Name:      "addport",
+				Usage:     "Add a new listener port (HOST_PORT:CONTAINER_PORT/PROTOCOL) to load balancer LOADBALANCER_NAME",
+				Action:    l.addport,
+				ArgsUsage: "LOADBALANCER_NAME HOST_PORT:CONTAINER_PORT/PROTOCOL",
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "certificate",
+						Usage: "Name of SSL certificate to use for port configuration (only required for https)",
+					},
+				},
+			},
+			{
+				Name:      "create",
+				Usage:     "Create a new load balancer LOADBALANCER_NAME in the environment specified in ENVIRONMENT_TARGET",
+				Action:    l.create,
+				ArgsUsage: "ENVIRONMENT_TARGET LOADBALANCER_NAME",
+				Flags: []cli.Flag{
+					cli.StringSliceFlag{
+						Name:  "port",
+						Value: &defaultPortFlag,
+						Usage: "Port configuration in format 'HOST_PORT:CONTAINER_PORT/PROTOCOL'",
+					},
+					cli.StringFlag{
+						Name:  "certificate",
+						Usage: "Name of SSL certificate to use for port configuration (only required for https)",
+					},
+					cli.BoolFlag{
+						Name:  "private",
+						Usage: "If specified, creates a private load balancer (default is public)",
+					},
+					cli.StringFlag{
+						Name:  "healthcheck-target",
+						Value: "TCP:80",
+						Usage: "Health check target in format 'PROTOCOL:PORT' or 'PROTOCOL:PORT/WITH/PATH'",
+					},
+					cli.IntFlag{
+						Name:  "healthcheck-interval",
+						Value: 30,
+						Usage: "Health check interval in seconds",
+					},
+					cli.IntFlag{
+						Name:  "healthcheck-timeout",
+						Value: 5,
+						Usage: "Health check timeout in seconds",
+					},
+					cli.IntFlag{
+						Name:  "healthcheck-healthy-threshold",
+						Value: 2,
+						Usage: "Number of consecutive successes required to count as healthy",
+					},
+					cli.IntFlag{
+						Name:  "healthcheck-unhealthy-threshold",
+						Value: 2,
+						Usage: "Number of consecutive failures required to count as unhealthy",
+					},
+				},
+			},
+			{
+				Name:      "delete",
+				Usage:     "Delete the load balancer LOADBALANCER_NAME",
+				ArgsUsage: "LOADBALANCER_NAME",
+				Action:    l.delete,
+			},
+			{
+				Name:      "dropport",
+				Usage:     "Drop the listener with host port HOST_PORT from load balancer LOADBALANCER_NAME",
+				Action:    l.dropport,
+				ArgsUsage: "LOADBALANCER_NAME HOST_PORT",
+			},
+			{
+				Name:      "get",
+				Usage:     "Describe load balancer LOADBALANCER_NAME",
+				Action:    l.read,
+				ArgsUsage: "LOADBALANCER_NAME",
+			},
+			{
+				Name:      "healthcheck",
+				Usage:     "View or update the health check of load balancer LOADBALANCER_NAME. If no options are specified, the health check is displayed.",
```

Comment:
[nit] this line wraps oddly for your average terminal screen, consider shorting the line. I think the 'if' statement is a bit redundant.